### ISSUE TITLE
[Accessibility] Add FAB doesn't work when talkback is enabled

### ISSUE THREAD

**Issue by criticalAY** [ankidroid/Anki-Android#20379]
### Checked for duplicates?

- [x] This issue is not a duplicate

### Does it also happen in the desktop version?

- [x] This bug does not occur in the latest version of Anki Desktop

### What are the steps to reproduce this bug?

- Enable TalkBack
- Trying adding a card

### Expected behaviour

Double tap opens the note editor or the other options that are intended

### Debug info

```text
on main branch
```

### Research

- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by criticalAY** [comment]
I want to verify the whole app for accessibility once but this was pointed out by one of our new contributors

**Comment by Mahakbajpai** [comment]
I want to work on this issue. Could you please assign it to me?
